mesheryctl/internal/cli/root/system/channel.go

```diff
@@ -222,7 +222,15 @@ var channelCmd = &cobra.Command{
 	Args:  cobra.NoArgs,
 	Example: `
 // Subscribe to release channel or version
-mesheryctl system channel [stable|stable-version|edge|edge-version]
+mesheryctl system channel 
+// To set the channel
+mesheryctl system channel set [stable|stable-version|edge|edge-version]
+// To pin/set the channel to a specific version
+mesheryctl system channel set stable-version 0.6.0
```

Comment:
Not apt :|

```suggestion
mesheryctl system channel set stable-v0.6.0-rc.6fc
```
This is the apt one

Reply:
sorry @alphaX86 I had not considered your example but an earler mesheryctl error which had redirected me to set the v as v0.6.0 so I had that in mind , 

Though I will make the rc one for now as you mentioned :+1: